packages/lesswrong/components/quickTakes/QuickTakesEntry.tsx

```diff
@@ -71,6 +71,32 @@ const styles = (theme: ThemeType) => ({
     marginRight: 8,
   },
   ...submitButtonStyles(theme),
+  commentForm: {
+    '& .form-input': {
+      margin: 0,
+      minHeight: 30
+    },
+    '& .ck.ck-editor__editable_inline': {
+      border: "none !important",
+    },
+    '& .EditorTypeSelect-select': {
+      display: 'none',
+    },
+  },
+  commentFormCollapsed: {
+    '& .EditorFormComponent-commentEditorHeight': {
+      minHeight: 'unset'
+    },
+    '& .EditorFormComponent-commentEditorHeight .ck.ck-content': {
+      minHeight: 'unset'
+    },
+    '& .LocalStorageCheck-root': {
+      display: 'none',
+    },
+    '& .ck .ck-placeholder': {
+      position: 'unset'
+    },
```

Comment:
These changes aren't forum-gated directly in the styles, but they're only used inside of the forum-gated functionality on L305.